src/NinjaTools.FluentMockServer/FluentAPI/Builders/FluentExpectationBuilder.cs

```diff
@@ -4,33 +4,29 @@
 using System.Net.Http;
 using JetBrains.Annotations;
 using NinjaTools.FluentMockServer.FluentAPI.Builders.HttpEntities;
+using NinjaTools.FluentMockServer.Models;
 using NinjaTools.FluentMockServer.Models.HttpEntities;
 using NinjaTools.FluentMockServer.Models.ValueTypes;
-// ReSharper disable UnusedAutoPropertyAccessor.Local
 
 namespace NinjaTools.FluentMockServer.FluentAPI.Builders
 {
        internal sealed class FluentExpectationBuilder : IFluentExpectationBuilder
         {
             [NotNull]
-            public static Models.Expectation Create(
-                [CanBeNull] HttpRequest httpRequest = null,
-                [CanBeNull] HttpResponse httpResponse = null,
-                [CanBeNull] HttpTemplate httpResponseTemplate = null,
-                [CanBeNull] HttpForward httpForward = null,
-                [CanBeNull] HttpTemplate httpForwardTemplate = null,
-                [CanBeNull] HttpError httpError = null,
-                [CanBeNull] Times times = null,
-                [CanBeNull] LifeTime timeToLive = null) => new Models.Expectation(httpRequest, httpResponse, httpResponseTemplate, httpForward, httpResponseTemplate, httpError, times, timeToLive);
+            public static Expectation Create(
+                HttpRequest? httpRequest = null,
+                HttpResponse? httpResponse = null,
+                HttpError? httpError = null,
+                Times? times = null,
+                LifeTime? timeToLive = null,
+                MockContext context = null) => new Expectation(httpRequest, httpResponse, httpError, times, timeToLive, context);
         
-            [CanBeNull] private HttpRequest HttpRequest { get; set; }
-            [CanBeNull] private HttpResponse HttpResponse { get; set; }
-            [CanBeNull] private HttpTemplate HttpResponseTemplate { get; set; }
-            [CanBeNull] private HttpForward HttpForward { get; set; }
-            [CanBeNull] private HttpTemplate HttpForwardTemplate { get; set; }
-            [CanBeNull] private HttpError HttpError { get; set; }
-            [CanBeNull] private Times Times { get; set; }
-            [CanBeNull] private LifeTime TimeToLive { get; set; }
+            private MockContext? Context { get; set; }
+            private HttpRequest? HttpRequest { get; set; }
+            private HttpResponse? HttpResponse { get; set; }
+            private HttpError? HttpError { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private set accessor in property 'HttpError'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)